source/fundamentals/time-series.txt

```diff
@@ -0,0 +1,107 @@
+=======================
+Time Series Collections
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn about **time series collections** in
+MongoDB, and how to interact with them in the MongoDB Kotlin driver.
+
+Time series collections efficiently store sequences of measurements over
+a period of time. Time series data consists of any data collected over
+time, metadata that describes the measurement, and the time of the
+measurement.
+
+.. list-table::
+   :widths: 33, 33, 33
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Example
+     - Measurement
+     - Metadata
+
+   * - Sales Data
+     - Revenue
+     - Company
+
+   * - Infection Rates
+     - Amount of People Infected
+     - Location
+
+Create a Time Series Collection
+-------------------------------
+
+To create a time series collection, pass the following parameters to the
+`createCollection() <{+api-kotlin+}/apidocs/mongodb-driver-kotlin-coroutine/mongodb-driver-kotlin-coroutine/com.mongodb.kotlin.client.coroutine/-mongo-database/create-collection.html>`__
+method:
+
+- The name of the new collection to create
+- The `TimeSeriesOptions <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/TimeSeriesOptions.html>`__ 
+  for creating the collection in a `CreateCollectionOptions <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/CreateCollectionOptions.html>`__ object
+
+.. literalinclude:: /examples/generated/TimeSeriesTest.snippet.create-time-series-collection.kt
+   :language: kotlin
+
+.. important::
+
+   Versions prior to MongoDB 5.0 cannot create a time series collection.
+
+To check if you successfully created the collection, send the
+``"listCollections"`` command to the `runCommand() <{+api-kotlin+}/apidocs/mongodb-driver-kotlin-coroutine/mongodb-driver-kotlin-coroutine/com.mongodb.kotlin.client.coroutine/-mongo-database/run-command.html>`__ method.
```

Comment:
Why not use the `listCollections` helper method on `MongoDatabase`?

Reply:
implemented. using this method not only made the code example in the docs easier to read, but also made the test simpler and the console output easier to read 💪